docs/dev/development.md

```diff
@@ -1,70 +1,90 @@
+First you need to create a _sandbox_ directory which will store the necessary configuration files and dependencies of [Cloudbreak Deployer](https://github.com/hortonworks/cloudbreak-deployer). This directory must be created outside of the cloned Cloudbreak git repository:
 ```
 mkdir cbd-local
+cd cbd-local
 ```
 
-To start the complete Cloudbreak ecosystem on your machine just execute the following sequence of commands:
+Add the following to the file `Profile` there under cbd-local (IPs could be different based on your network/virtualbox setup):
 ```
-cd cbd-local
-curl https://raw.githubusercontent.com/sequenceiq/cloudbreak-deployer/master/install | sh && cbd --version
-cbd update master
-cbd init
-cbd start
+export PUBLIC_IP=192.168.99.100
+export PRIVATE_IP=192.168.99.100
+export CB_LOCAL_DEV_BIND_ADDR=192.168.99.1
+export DOCKER_MACHINE=default
+export ULU_SUBSCRIBE_TO_NOTIFICATIONS=true
+export CB_INSTANCE_UUID=$(uuidgen | tr '[:upper:]' '[:lower:]')
+export CB_SCHEMA_SCRIPTS_LOCATION=/Users/YOUR_USERNAME/YOUR_PROJECT_DIR/cloudbreak/core/src/main/resources/schema
+export UAA_DEFAULT_USER_PW=YOUR_PASSWORD
 ```
-If everything went well then the Cloudbreak will be available on http://192.168.59.103:3000. For more details and config parameters please check the documentation of [Cloudbreak Deployer](https://github.com/sequenceiq/cloudbreak-deployer).
 
-The deployer has generated a `certs` directory under `cbd-local` directory which will be needed later on to set up the IDEA properly.
+The CB_SCHEMA_SCRIPTS_LOCATION environment variable configures the location of SQL scripts that are in the 'core/src/main/resources/schema' directory in the cloned Cloudbreak git repository. Please note that the full path needs to be configured and env variables like $USER cannot be used. You also have to set a password for your local Cloudbreak in UAA_DEFAULT_USER_PW.
 
-The next step is to edit the `cbd-local/Profile` file with any editor and add the CB_SCHEMA_SCRIPTS_LOCATION environment variable which configures the location of SQL scripts that are in the 'core/src/main/resources/schema' directory in the cloned Cloudbreak git repository. Please note that the full path needs to be configured and env variables like $USER cannot be used.
+To start the complete Cloudbreak ecosystem on your machine just execute the following sequence of commands:
+```
+curl https://raw.githubusercontent.com/hortonworks/cloudbreak-deployer/master/install-dev | sh [-s branch] && cbd --version
+```
+Use the -s branch option for sh here in case you'd like to checkout another branch than master.
 
 ```
-export CB_SCHEMA_SCRIPTS_LOCATION=/Users/myusername/prj/cloudbreak/core/src/main/resources/schema
+docker-machine start default
```

Comment:
The name of the default instance is `cbd` so the right command is `docker-machine start cbd`